pkg/collector/corechecks/cluster/kubernetes_apiserver.go

```diff
@@ -36,6 +36,7 @@ type KubeASCheck struct {
 	lastWarnings          []error
 	instance              *KubeASConfig
 	KubeAPIServerHostname string
+	eventResVersion       string
```

Comment:
- Let's prefix the name by `latest` to make the intention clear.
- As the `APIClient.LatestEvents` doc does not mention the token is a `resVersion`, that implementation detail does not need to bleed into the check.

I'd go with `latestEvent` or `latestEventToken`, WDYT?

Reply:
yeah good point